core/amber/src/main/scala/edu/uci/ics/amber/engine/common/ambertag/neo/VirtualIdentity.scala

```diff
@@ -0,0 +1,19 @@
+package edu.uci.ics.amber.engine.common.ambertag.neo
+
+// The following pattern is a good practice of enum in scala
+// We've always used this pattern in the codebase
+// https://pedrorijo.com/blog/scala-enums/
+
+trait VirtualIdentity
+
+object VirtualIdentity {
+
+  trait ActorVirtualIdentity extends VirtualIdentity
+
+  case class NamedActorVirtualIdentity(name: String) extends ActorVirtualIdentity
```

Comment:
why call it named actor? In akka I thought every actor has a name. I think workers name is their getGlobalIdentity(). Explain with a comment.

Reply:
I think `WorkerActorVirtualIdentity` should be a better name so I renamed it.